CODEOWNERS

```diff
@@ -0,0 +1 @@
+/jenkins/	@jlowe @revans2 @tgravescs @GaryShen2008
```

Comment:
This file needs to be in the `.github/` directory rather at the top-level.  Also should Tim be in this list since he wrote a significant chunk of it?

Reply:
The file can be in either .github or root, per https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-file-location .  I am ok if we prefer .github, let me know and I can change.  

Adding nvtimliu to the list.

Reply:
Yes, please put in .github so we don't clutter the root directory anymore than it already is.

Reply:
Moved to .github